### ISSUE TITLE
Update on OA system, and Durafet pH sensor Malfunctioning

### ISSUE THREAD

**Issue by laurahspencer** [sr320/LabDocs#467]
@yaaminiv I got the OA system running today, and left both header tanks with a set point of 7.45-7.55, to see if they can hold at pH 7.5. There are Durafets in either header tank, and one in a culture tank. One Durafet, however, is showing an error code, blinking "100" and "error input pv 1".  I did a quick search in user manuals for both the Durafet and the Honeywell control panel, and didn't see suggestions for trouble shooting.  We should do some research/planning in the next few days, so we can try to trouble shoot on Wednesday. Here is the info for the instruments:  

Honeywell Durafet model number:  51453503-505; I think this is the [user manual](https://www.honeywellprocess.com/library/support/Public/Documents/70-82-25-115.pdf) 

[Honeywell  Analyzer-UDA2182 Multiple Input](https://www.honeywellprocess.com/en-US/explore/products/instrumentation/analytical-instruments-and-sensors/dissolved-oxygen/Pages/uda2182-multiple-input-analyzer.aspx) control panel (I think), and [user manual](http://www.argus-seelaus.com/UDA2182%20Manual.pdf)

One plan of attack would be to try to clean it as per the manual's instructions, using dilute HCl, q-tip, etc. I don't think it's a wiring issue, because the temperature is reading OK.

**Comment by Ellior2** [comment]
You might want to disconnect the Durafet from the cable and look for corrosion in the connection. I have also tried just unplugging and replugging in the 2182 when I get weird messages and that works. Also if you push the middle button (I think it's "enter"), it will ask you if the probes were replaced, and if you select "yes" it does a reset to manufacturer settings.

**Comment by sr320** [comment]
Thanks! This is very helpful.

**Comment by Ellior2** [comment]
We've definitely had corrosion issues in the past. That connection really needs to be protected against saltwater splashing of any kind. Unfortunately you also have to have the probe submerged at least 2-3 inches which makes the threading right above the sensor pretty useless. Is there a photo?

**Comment by laurahspencer** [comment]
Thanks, Rhonda!  No photo at this point; the Durafets are submerged in water during use, so they have a protective housing. It's possible that there was a leak.  I will try your reset idea.  By "2182" do you mean the Honeywell Control Panel?

**Comment by Ellior2** [comment]
Yes the Honeywell UDA2182 control panel. I have never seen protective housing for the Durafets. Is that something you guys made? I'm curious. Anyways, you should be able to see the corrosion if that's the issue.

**Comment by laurahspencer** [comment]
These are how our Durafets are housed:

**Comment by sr320** [comment]
![pi](https://safsoa.files.wordpress.com/2016/01/img_3940.jpeg?w=1080)

**Comment by laurahspencer** [comment]
![Durafets](https://github.com/laurahspencer/O.lurida_Stress/blob/master/Images/IMG_6740.JPG?raw=true)

**Comment by Ellior2** [comment]
So this is what they normally look like without those housings. I'd check inside and disconnect. See those vertical bands on the middle probe? That is where I have seen corrosion which led to similar error messages you describe.

![image](https://cloud.githubusercontent.com/assets/20071030/22664624/52d8362c-ec66-11e6-86d6-a2ae07455dec.png)

**Comment by laurahspencer** [comment]
Ok- good to know- thanks. When you have found corrosion in the past, was did you have any success cleaning/recovering the probe?

**Comment by Ellior2** [comment]
Yes I have, I just wipe it off with a paper towel and make sure it is VERY dry. Do you guys have backups? They have been known to take up to 4 months to ship (due to high demand, back ordering) so we always keep a set of backups. I would order more if you don't have any spare, especially if that is how you are controlling your treatments. They can very randomly fail. We typically replace ours once a year, whether or not it still seems like its working.

**Comment by laurahspencer** [comment]
Results: 
  * We tried both unplugging and resetting the problem Durafet; neither solved the issue. 
  * Uncapped the probe, and found no corrosion. 
  * When plugged in, the temperature reading was ~accurate/functional, but the pH reading was showing error.  
  * We determined that there is currently nothing to do to fix this probe, and it is now out of commission, stored in its original box in the hatchery dry lab.
